Makefile

```diff
@@ -54,4 +57,17 @@ stage:
 
 DIALYZER_APPS = kernel stdlib erts sasl eunit syntax_tools compiler crypto
 
+##
+## deploy to CCM
+##
+deploy:rel
+	scp /lib/x86_64-linux-gnu/libtinfo.so* $(CCM):~/
```

Comment:
you set it to the IP address where you want to push the updated beam too, so `make CCM=core@10.134.32.32`